internal/services/policy/remediation_management_group.go

```diff
@@ -0,0 +1,251 @@
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/policyinsights/mgmt/2019-10-01-preview/policyinsights"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/tf"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	managmentGroupValidate "github.com/hashicorp/terraform-provider-azurerm/internal/services/managementgroup/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/policy/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/policy/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/suppress"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+func resourceArmManagementGroupPolicyRemediation() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceArmManagementGroupPolicyRemediationCreateUpdate,
+		Read:   resourceArmManagementGroupPolicyRemediationRead,
+		Update: resourceArmManagementGroupPolicyRemediationCreateUpdate,
+		Delete: resourceArmManagementGroupPolicyRemediationDelete,
+
+		Importer: pluginsdk.ImporterValidatingResourceId(func(id string) error {
+			_, err := parse.ResourcePolicyRemediationID(id)
+			return err
+		}),
+
+		Timeouts: &pluginsdk.ResourceTimeout{
+			Create: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Read:   pluginsdk.DefaultTimeout(5 * time.Minute),
+			Update: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Delete: pluginsdk.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*pluginsdk.Schema{
+			"name": {
+				Type:         pluginsdk.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.RemediationName,
+			},
+
+			"management_group_id": {
+				Type:         pluginsdk.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: managmentGroupValidate.ManagementGroupID,
+			},
+
+			"policy_assignment_id": {
+				Type:     pluginsdk.TypeString,
+				Required: true,
+				// TODO: remove this suppression when github issue https://github.com/Azure/azure-rest-api-specs/issues/8353 is addressed
+				DiffSuppressFunc: suppress.CaseDifference,
+				ValidateFunc:     validate.PolicyAssignmentID,
+			},
+
+			"location_filters": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				Elem: &pluginsdk.Schema{
+					Type: pluginsdk.TypeString,
+				},
+			},
+
+			"policy_definition_id": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				// TODO: remove this suppression when github issue https://github.com/Azure/azure-rest-api-specs/issues/8353 is addressed
+				DiffSuppressFunc: suppress.CaseDifference,
+				ValidateFunc:     validate.PolicyDefinitionID,
+			},
+
+			"resource_discovery_mode": {
```

Comment:
is resource really needed? 
```suggestion
			"discovery_mode": {
```

Reply:
I believe so! It's a specific callout for how we'll find which resources to remediate under the umbrella of each type of remediation policy.